### PR TITLE
1309 intra angle

### PR DESCRIPTION
**Pull request by Danbr4d**
Set up the initial intra angle module

### CODE REVIEW

**Comment by Danbr4d** [comment]
Not sure how to set up the tests for this I've looked at IntraDistance but I'm not sure where the data comes from

File: src/modules/intraangle/gui/intraanglewidget.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    void updateControls(const Flags<ModuleWidget::UpdateFlags> &updateFlags = {}) override;
+
+    /*
+     * Widgets / Functions
+     */
+    private:
+    // Set data targets in graphs
+    void setGraphDataTargets(IntraAngleModule *module);
+
+    private slots:
```
Comment (rprospero) [inline src/modules/intraangle/gui/intraanglewidget.h:46]: I don't believe we need a slots section if we have no slots.

File: src/modules/intradistance/functions.cpp
Comment (rprospero) [inline src/modules/intradistance/functions.cpp:1]: I might be a bit dense here, but do we need this file?  It seems like we're not adding any actual code.

**Review by rprospero — APPROVED** [review]
I had a couple of questions, but I'm happy to approve this as long as someone has answers.

File: src/modules/intradistance/functions.cpp
Comment (trisyoungs) [reply src/modules/intradistance/functions.cpp:1]: Nope! This is in my intra-rdf PR that this one is based off.  I'll get rid of it there.

**Pull request merged**